.github/workflows/blueprint.yml

```diff
@@ -48,29 +48,23 @@ jobs:
       - name: Get Mathlib cache
         run: ~/.elan/bin/lake exe cache get || true
 
-      - name: Build project
-        run: ~/.elan/bin/lake build equational_theories
-
-      - name: Cache mathlib API docs
+      - name: Cache build artifacts and Mathlib API docs
         uses: actions/cache@v4
         with:
           path: |
-            .lake/build/doc/Init
-            .lake/build/doc/Lake
-            .lake/build/doc/Lean
-            .lake/build/doc/Std
-            .lake/build/doc/Mathlib
-            .lake/build/doc/declarations
-            .lake/build/doc/find
-            .lake/build/doc/*.*
+            .lake/build
+            !.lake/build/doc-data
```

Comment:
I'd say we should exclude the mathlib directory therein as well, it's very large and probably faster to fetch via `lake exec cache get`

Reply:
Which is the exact path you're referring to? `.lake/build/doc/Mathlib`?

Reply:
At the phone right now. I think the path is something like `.lake/packages/mathlib` or so - look at your local `.lake` directory :-)

`batteries` can also be excluded.

Reply:
But those are already excluded, right? They're not within the `.lake/build` directory so it's not supposed to be cached by this configuration.

Reply:
Ah, I missed that! Nevermind then.

(I wonder if we should include libraries we depend upon that are not mathlib or dependend upon by mathlib, like egg. But we can iterate!)

Reply:
Ok, I would merge for now.